README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
+right turn and *cough*, *splutter* &mdash; their car chokes to a stop. At once
+their mind changes to being a _mechanic_ or a _developer_. They think about
 petroleum, spark-plugs and ignitions. They can _debug_ their car using
 techniques they've learned. They can imagine what might be wrong. To get more
 data or to verify their guess, they _open up the hood_. Learning to be a
 developer (_computer mechanic_?) is similar. You must start "looking under the
 hood" of your computer. To access the "hood's" contents, you must use the
-shell.
+terminal.
 
-Developers "talk" to the computer by engaging it at the shell level. The shell
+Developers "talk" to their computers through the terminal. The terminal
 is just another way to collaborate with the computer. It might feel
 "old-school" or clunky at first compared with Windows or MacOSX. As you learn
 more, though, it might surprise you!
 
-Shell skills are critical in being a developer or HTML author.  Comfort in the
-shell is an assumption for many technical jobs.
+Terminal skills are critical in being a developer or HTML author.  Comfort in the
+terminal is an assumption for many technical jobs.
 
 ## Learning Objectives
 
-1. Define what a shell program is
+1. Define what a terminal program is
 2. Recognize a shell interface
-3. Recognize operating systems providing a shell
+3. Recognize operating systems providing a terminal
 
-## Define What a Shell Program Is
+## Define What a Terminal Program Is
 
-The shell is the program on your computer that takes in commands you type and
-takes action based on those commands. You can think of yourself as having a
-conversation with the computer.
+### Advice
 
-When you see a shell it will be at a "command prompt." This means, from the
-computer's perspective "I'm listening..." Prompts may look different from
-computer to computer. Customized prompts express developer uniqueness. We've
-seen weather reports, beautiful colors, computer temperatures in prompts. At
-heart, though, they they all carry your command to the operating system and run
-it.
+First of all, we have to be honest here: the difference between "shell" and
+"terminal" is quite easy to get confused by. To make things worse, developers
+are often imprecise and say one to mean the other and vice versa.
 
-You can ask the computer to do many types of tasks. From the command prompt you
-can move or rename files just like you would do from a window environment.  You
+In order to try to help build your instinct for understanding the concepts,
+we'll often "over-explain" or make a sentence seem ridiculously long. We're
+doing this intentionally in order to try to help you build an intuition about
+the actors in this story.
+
+Also, to help keep terminology from getting in the way, we involve genies.
+We admit, this is a whimsical approach but assure you no genies were harmed in
+the production of this lesson.
+
+Be patient with yourself as you learn this material.
```

Comment:
So one of the metaphors that I really love is starting to think of experience as analogous to video game levels. So I'm not 36 years old, I'm level 36. I've not been developing and teaching for 8 years, I'm a level 8 developer. I find that this takes a lot of the personal feeling out of the whole "I'm never going to be a developer", and helps people readjust their expectations. There might be space for something like this here? It's a suggestion, not a requirement, but it helps to develop patience. It might also be something that is better delivered live (I used to use it during study groups).